Contributing reorg [Fixes #1103 + #1247] (#1248)
ethereum/ethereum-org-website: src/content/about/index.md

---
title: About Us
description: About the team, community and mission of ethereum.org
lang: en
sidebar: true
---

# About ethereum.org {#about-ethereumorg}

> ethereum.org is a public, open-source resource for the Ethereum community that anyone can contribute to. We have a small team dedicated to maintaining and developing the site which is funded by the [Ethereum foundation](/foundation/).

Reviewer: ```suggestion
ethereum.org is a public, open-source resource for the Ethereum community that anyone can contribute to. We have a small team dedicated to maintaining and developing the site which is funded by the [Ethereum Foundation](/foundation/).
```

> ethereum.org is a public, open-source resource for the Ethereum community that anyone can contribute to. We have a small team dedicated to maintaining and developing the site which is funded by the [Ethereum foundation](/foundation/).

Author: +100 on the consistency. TBH I've tried to roll with ethereum.org (lowercase). There seemed to be a mix in the all the docs I went through. Happy to go with team consensus on that though.

## Our vision

### ethereum.org's mission is to be the best portal for Ethereum's growing community

We're an educational resource, designed to help new users become familiar with Ethereum and its key concepts. We want to:

- explain Ethereum to anyone new to the technology
- help new users get started with ETH and Ethereum
- help new developers to start building
- cover updates in the Ethereum world
- showcase resources created by the community
- bring Ethereum education to as many languages as possible

We have some core principles that help us do this.

## Core principles

### 1. ethereum.org is a portal to Ethereum 🌏

> We want our users to have their interest piqued and their questions answered. So our portal needs to combine information, "magic moments" and links to the brilliant community resources that exist out there. The purpose of our content is to be an “on-boarding portal” and not a substitute for the extensive resources that already exist. We're keen to support and integrate with community built resources, giving them more visibility and making them more discoverable.

Reviewer: ```suggestion
We want our users to have their interest piqued and their questions answered. So our portal needs to combine information, "magic moments" and links to the brilliant community resources that exist out there. The purpose of our content is to be an “onboarding portal” and not a substitute for the extensive resources that already exist. We're keen to support and integrate with community built resources, giving them more visibility and making them more discoverable.
```

In the /adding-products/ page we use "onboarding". I suggest we stick with one.

> Ethereum's community is at the heart of this: we need to not just serve the community, but incorporate their feedback and work with a large group of stakeholders to improve it. The website isn't just for the community we have now but must anticipate how it will grow in the future. Ethereum is global, containing people from many languages, regions, and cultures.

Reviewer: ```suggestion
[Ethereum's community](https://ethereum.org/en/community/) is at the heart of this: we need to not just serve the community, but incorporate their feedback and work with a large group of stakeholders to improve it. The website isn't just for the community we have now but must anticipate how it will grow in the future. Ethereum is global, containing people from many languages, regions, and cultures.
```

Good spot to link to the Community page?

Author: Good spot. On reading this again, the sentences are a little clunky here so I'm going to re-write it a little.

> Ethereum's community is at the heart of this: we need to not just serve the community, but incorporate their feedback and work with a large group of stakeholders to improve it. The website isn't just for the community we have now but must anticipate how it will grow in the future. Ethereum is global, containing people from many languages, regions, and cultures.

Reviewer: ```suggestion
Ethereum's community is at the heart of this: we need to not just serve the community, but incorporate their feedback and work with a large group of stakeholders to improve it. The website isn't just for the community we have now; we must anticipate how it will grow in the future. Ethereum is global, containing people from many languages, regions, and cultures.
```
Not sure but "The website isn't just for the community we have now but must anticipate how it will grow in the future." sounds a bit awkward.
